### DISCUSSION TITLE
상호작용을 만들고싶습니다만............,,

### DISCUSSION THREAD

**Discussion by peullea** [naverz/zepeto-studio-kor#532] [5. Q&A]
일단 저는 코딩을 하나도 못합니다. 그래서 가이드와 검색을 통해 알아낸 정보로 진행하고있습니다.

1. UI가이드를 따라 버튼을 만드는거까지는 성공하였으나(여기> https://studio.zepeto.me/kr/guides/ui-events )
가까이 가면 버튼이 나타나는것도 안되고 (Canvas를 World Space로 바꾸면 아무것도 안되더라고요. 코드를 적용해도 말이죠..ㅠ
2. 앉는 애니메이션은 당최 어떻게 만드는지 잘 모르겠습니다.

저도 검색을 안해본 건 아닙니다만 제가 뭘 잘못했는지(도 모르지만) 코드를 짜는데 오류가 한두개씩 나더군요..ㅠ
3. 혹시 멀티플레이 서버부터 구축을 해야하는 문제인가요?
 캐릭터 컨트롤러를 일단은 https://studio.zepeto.me/kr/guides/character-control 를 통해 만들었습니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. 답변 드립니다.

1. world space Canvas 관련 설정은 아래 질문에 설정 방법 및 코드를 올려두었습니다. 
https://github.com/naverz/zepeto-studio-kor/discussions/442#discussioncomment-2263512

2. 앉는 애니메이션은 Packages > ZEPETO.Character.Controller > Runtime > _Resources > Animations 에서 원하는 것과 유사한 제스쳐를 찾아 보세요.
애니메이션 예제 코드는 아래 링크를 참고하여 주세요.
https://github.com/naverz/zepeto-studio-kor/discussions/496#discussioncomment-2362863

3. 멀티플레이를 구축하지 않아도 상호작용은 가능합니다.

**Reply by peullea** [reply]
감사합니다. 다시 해보겠습니다 ㅠㅠ

**Reply by hipsc159** [reply]
다른 플레이어들과 애니메이션 동기화는 어떻게 하나요?
